.github/ISSUE_TEMPLATE/performance_problem.yml

```diff
@@ -0,0 +1,64 @@
+name: Performance problem
+about: Report performance problems or areas of concern
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before reporting performance problems here, please make sure you are on the latest version of Paper.
+        Forks of Paper receive no support here. If you are using a fork, please make sure that this problem also affects Paper.
+  - type: input
+    attributes:
+      label: Timings or Profile link
+      description: We ask that all timings/profiles are a link, not a screenshot. Screenshots inhibit our ability to figure out the real cause of the issue.
+      placeholder: "Example: https://timings.aikar.co/?id=6b48586fbbdd48e585ca0ebb07c59dd0"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Description of issue
+      description: If applicable, please describe your issue.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Plugin list
+      description: All plugins running on the server.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Server config files
+      description: We need bukkit.yml, spigot.yml, paper.yml, and server.properties. You can paste it below or use a paste site like https://paste.gg.
+      value: |
+        ```
+        Paste configs or paste.gg link here!
+        ```
+      placeholder: Please don't remove the backticks; it makes your issue a lot harder to read!
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Paper version
+      description: |
+        Paste the full, unmodified output of `/version` on your server WITH the Minecraft version.
```

Comment:
This wording is a bit confusing?

Reply:
That's how it was in the original template:
https://github.com/PaperMC/Paper/blob/master/.github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md#paper-version

I added "full, unmodified" either by request or on my own. How should it be reworded?

Reply:
Maybe: "Run /version on your server and post the full, unmodified output here. "

Maybe we could even include an example? Can you use spoilers in those descriptions? 
```
> version
[20:34:42 INFO]: This server is running Paper version git-Paper-539 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
[20:34:42 INFO]: Checking version, please wait...
[20:34:42 INFO]: Previous version: git-Paper-"656a74452" (MC: 1.16.5)
[20:34:42 INFO]: You are 1 version(s) behind
```

Reply:
Added to all 3 files